lib/percy/common/engine.rb

```diff
@@ -0,0 +1,7 @@
+module Percy
+  module Common
+    class Engine < ::Rails::Engine
```

Comment:
I'm wondering how this project has access to Rails? It's not in the `gemspec` and `percy-hub` doesn't require Rails.